modin/pandas/base.py

```diff
@@ -40,6 +40,14 @@
 sentinel = object()
 
 
+def is_scalar(obj):
```

Comment:
I think it's better to move this function into `modin/pandas/utils.py`.
Also, we could use this function everywhere we use `pandas.is_scalar` and delete importing `pandas.is_scalar`. Or we need to change importing `is_scalar` from pandas to `pandas_is_scalar` in all files in order to avoid confusing whose is the function (Modin or pandas).

Reply:
You're right. I moved it to utils and used everywhere instead of Pandas version.